heat/core/tests/test_random.py

```diff
@@ -310,6 +348,25 @@ def test_randn(self):
         self.assertFalse(np.allclose(a, c))
         self.assertFalse(np.allclose(b, c))
 
+    def test_randperm(self):
+        state = torch.random.get_rng_state()
+
+        # results
```

Comment:
can you test with float64 as well?

also should the return values should be `int64` correct? Can you test to make sure that they are what is expected?